.github/workflows/docker-image.yml

```diff
@@ -8,21 +7,31 @@ on:
     paths-ignore: [ 'README.md']
 
 jobs:
-
-  build:
-
+  job-one:
+    name: job-one
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v3
-    - name: Build the Docker image
-      run: docker build --target test --tag my-test-image .
-    - run: docker run --env-file .env.test my-test-image tests   
-    - run: docker run -e TRELLO_KEY=${{ secrets.TRELLO_KEY }} -e TOKEN=${{ secrets.TOKEN }} my-test-image tests_e2e
-
-
-
-
-
-
-
+      - uses: actions/checkout@v3
+      - name: Build the Docker image
+        run: docker build --target test --tag my-test-image .
+      - run: docker run --env-file .env.test my-test-image tests   
+      - run: docker run -e TRELLO_KEY=${{ secrets.TRELLO_KEY }} -e TOKEN=${{ secrets.TOKEN }} my-test-image tests_e2e
+
+  job-two:
+    name: job-two
+    runs-on: ubuntu-latest
+    needs: job-one
```

Comment:
Calling the first step "test" or something might be a little clearer, but I agree this gets the job done!